server/plugin.go

```diff
@@ -314,6 +308,15 @@ func (p *Plugin) start(isRestart bool) {
 		p.syncUserJob = job
 	}
 
+	// Unregister and re-register slash command to reflect any configuration changes.
+	if err = p.API.UnregisterCommand("", "msteams"); err != nil {
+		p.API.LogWarn("Failed to unregister command", "error", err)
+	}
+
+	if err = p.API.RegisterCommand(p.createCommand(p.getConfiguration().SyncLinkedChannels)); err != nil {
+		p.API.LogError("Failed to register command", "error", err)
+	}
+
```

Comment:
I'm assuming relocating this relates to the testing changes, but do we understand what was preventing this from "running normally" before?

Reply:
I think the problem was a race condition (The check certificates was taking time, so the plugin wasn't registering it), I can go back to the other way because now I should be replying to the list of certificate really quickly because is mocked, and if not, it is at least not forwarded to Microsoft api, so it shouldn't be a problem now.